docs/CONTRIBUTING.md

```diff
@@ -129,13 +130,14 @@ yarn package
 > If you need your builds to support older distros, run the build commands
 > inside a Docker container with all the build requirements installed.
 
-### Test
+## Test
 
-There are three kinds of tests in code-server:
+There are four kinds of tests in code-server:
 
 1. Unit tests
-2. Integration tests
-3. End-to-end tests
+2. Script tests
+3. Integration tests
+4. End-to-end tests
```

Comment:
**non-blocking(nice-to-have)** it's typically less error-prone to just use `1.` for all of them and let markdown convert them into 1 2 3 4 etc.

Reply:
I wish I could remember the point, but @code-asher mentioned a point _against_ doing this so I haven't, but yeah, it probably is less error-prone!

Reply:
Ahaha yeah because I read these in my editor and all I see is 1 1 1 11 1 1 1 1 1 1111 :stuck_out_tongue: 

Really this should not be done manually, I would use a VS Code plugin or something to number them automatically.